test/util_test.dart

```diff
@@ -13,22 +13,48 @@
 // limitations under the License.
 
 @TestOn('vm')
+import 'dart:io';
+
+import 'package:analyzer/dart/analysis/results.dart';
 import 'package:analyzer/dart/analysis/utilities.dart';
-import 'package:analyzer/dart/ast/token.dart';
 import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:over_react_codemod/src/constants.dart';
 import 'package:over_react_codemod/src/executables/dependency_validator_ignore.dart';
-
+import 'package:over_react_codemod/src/react16_suggestors/react16_utilities.dart';
+import 'package:over_react_codemod/src/util.dart';
 import 'package:path/path.dart' as p;
 import 'package:pub_semver/pub_semver.dart';
 import 'package:source_span/source_span.dart';
 import 'package:test/test.dart';
 
-import 'package:over_react_codemod/src/constants.dart';
-import 'package:over_react_codemod/src/react16_suggestors/react16_utilities.dart';
-import 'package:over_react_codemod/src/util.dart';
-
 void main() {
   group('Utils', () {
+    group('isA', () {
+      late ResolvedUnitResult result;
+      CompilationUnit astNode;
+
+      setUp(() async {
+        // final sourceFile = SourceFile.fromString(content)
+        // astNode = parseString(content: content).unit;
+        var path = p.join(Directory.current.absolute.path, 'test',
+            'util_test_things_to_parse.dart');
```

Comment:
It's hard to set up a resolved AST on a string in memory, so I made a separate file.

Reply:
FYI there's `SharedAnalysisContext` which makes it a bit easier. 
Example: 
- Setup: https://github.com/Workiva/over_react_codemod/blob/5e30958979a650d0850fadabb068f6ea17d542a7/test/util/element_type_helpers_test.dart#L57-L67
- Usage: https://github.com/Workiva/over_react_codemod/blob/5e30958979a650d0850fadabb068f6ea17d542a7/test/util/element_type_helpers_test.dart#L87-L106